tests/test_cli.py

```diff
@@ -15,6 +16,7 @@
 
 TEST_DIR = os.path.dirname(__file__)
 TEST_AUDIO_DIR = os.path.join(TEST_DIR, 'data', 'audio')
+TEST_IMAGE_DIR = os.path.join(TEST_DIR, 'data', 'image')
```

Comment:
Aren't we missing the same for video as well?

Reply:
Ah I included the video in the image folder, but I'll move it to a video specific folder.